src/utils/plotting/simple_plots.jl

```diff
@@ -66,3 +65,16 @@ end
     end
     t.vp[1]
 end
+
+
+# Auxiliary function for annotation
+function text_in_set_plot!(fig, po::Polyhedra.Rep, t)
```

Comment:
I don't know where this is used, but it could be incorporated in a recipe as well with an optional argument
Interestingly recipees are not only used to define a default plot for a structure, but also to define other type of plotting functions, but where one can still use all the classical plotting arguments   

See "Plot Recipes" and not "Type Recipes" here https://docs.juliaplots.org/latest/recipes/

Reply:
I used it for the Benoît's example of gol lazar belta. Thanks for the comment ! I will merge now 
and see later how to do that.

Reply:
But indeed, could be added as a plot parameters, even for hypperectangle, if for example you want to add a text in its center